src/schema/rules/checks/hints.yaml

```
@@ -0,0 +1,26 @@
+---
+SuspiciouslyLongDesign:
```

```suggestion
SuspiciouslyLongBOLDDesign:
```

These are BOLD-specific. I don't think supporting other functional imaging modalities (e.g., EEG) is feasible in this rule, but I think the rule name should reflect its limited scope.